backend/src/model/dto/plantings.rs

```diff
@@ -116,4 +126,6 @@ pub struct PlantingSearchParameters {
     pub plant_id: Option<i32>,
     /// The id of the plants layer the planting is placed on.
     pub layer_id: Option<i32>,
+    /// A date where the plantings should exists relative to.
```

Comment:
This is somewhat confusing? Do we fetch plants before the date, after the date or around the date?
Maybe it would be better to use 2 parameters instead so the frontend can control what to fetch (start_date and end_date)?